regression/pages/lms/__init__.py

```diff
@@ -12,3 +12,5 @@
 LOGIN_BASE_URL = 'https://{}:{}@courses.stage.edx.org'.format(
     BASIC_AUTH_USERNAME, BASIC_AUTH_PASSWORD)
 BASE_URL = os.environ.get('test_url', LOGIN_BASE_URL)
+
+BASE_URL_LMS = 'https://stage.edx.org'
```

Comment:
@benpatterson 
Sorry for my confusion, I am not getting this point actually. This is the html link we are going to click:
`self.q(css='a[href="https://stage.edx.org/edx-terms-service"]').click()`
Making 
`BASE_URL_LMS = https://stage.edx.org` will click the link and our test would work but if we change it to CMS `https://studio.stage.edx.org` the click won't be initiated.
What I mean is:
`self.q(css='a[href="https://stage.edx.org/edx-terms-service"]').click()` - This would click
`self.q(css='a[href="https://studio.stage.edx.org/edx-terms-service"]').click()` - This won't and will fail the test.